ci/OWNERS

```diff
@@ -302,6 +302,7 @@ pkgs/development/python-modules/buildcatrust/ @ajs124 @lukegb @mweinelt
 
 # Bazel
 /pkgs/development/tools/build-managers/bazel @Profpatsch
+/pkgs/by-name/ba/bazel_8 @Profpatsch
```

Comment:
Let's not burden people with more pings unless they confirm being okay with it. @Profpatsch do you even still want to help maintain Bazel? If not, you should probably be removed from here.

And independent of whether @Profpatsch stays listed here, I'm suggesting you add yourself @boltzmannrain, since you've done so much great Bazel maintenance recently :D